server/controllers/finance/reports/operating/index.js

```diff
@@ -23,31 +18,22 @@ const INCOME_ACCOUNT_TYPE = 4;
 const DECIMAL_PRECISION = 2; // ex: 12.4567 => 12.46
 
 /**
- * @description this function helps to get html document of the report in server side
- * so that we can use it with others modules on the server side
+ * @function reporting
+ *
+ * @description
+ * Renders the Compte d'Exploitation
+ *
  * @param {*} options the report options
  * @param {*} session the session
  */
-function reporting(opts, session) {
-  const params = opts;
-  let docReport;
-  const options = _.extend(opts, {
+async function reporting(params, session) {
```

Comment:
Do you think it's a good idea to use this async without a ```try catch```?

Reply:
The try/catch occurs in the function that calls this one - on this line: https://github.com/IMA-WorldHealth/bhima/pull/4487/files#diff-4c6917cb3623fce5a08ff0fa7b067112R153

Reply:
okay